std_msgs/Header header
sensor_msgs/PointCloud2 cloud1
sensor_msgs/PointCloud2 cloud2
sensor_msgs/PointCloud2 cloud3
sensor_msgs/PointCloud2 cloud4
